### ISSUE TITLE
[deleted]

### ISSUE THREAD

**Issue by Podberezovik34** [popcorntime/popcorntime#2870]

**Comment by ivan1986** [comment]
I can also add about some huilo, I’m glad that your ass is on fire..
Holy crap, Kremlinbots on GitHub, it was only worse when the tramp-asses was in the bots’ user agents.

**Comment by Ravshanovich** [comment]
Позиция нацистов истребляющих своих же граждан, Давай побольше лозунгов нацист.

**Comment by Ravshanovich** [comment]
> This is the position of the Russian API administrator (who is also a dev here, and personally tbh is my position too.

He's not Russian, he's clearly a Ukrainian rat.

**Comment by dutchy1001** [comment]
> Closing this issue but everyone can talk freely below.

Guess this was taken **TO** literally

**Comment by merseyV** [comment]
You can't nuke! in GitHub then?

**Comment by merseyV** [comment]
Nah I'm for nuke it!! 🤣🤣🤣

**Comment by dutchy1001** [comment]
no point deleting, just ignoring will do me (after this post).
Politics don't belong in Github

**Comment by merseyV** [comment]
lol ... now you know why I'm not mod. material? Too much power and I'd use it ruthlessly, without restraint.

I'm a lot wiser than you two think?

**Comment by merseyV** [comment]
lol ... now you know why I'm not mod. material? Too much power and I'd use it ruthlessly, without restraint.

[Man's got to know his limitations?](https://www.youtube-nocookie.com/embed/uki4lrLzRaU?playlist=uki4lrLzRaU&autoplay=1&iv_load_policy=3&loop=1&start=136) 😆

**Comment by Ravshanovich** [comment]
@kiriles90     For morons, I explain that this video was recorded 4 years ago when the Minsk Agreement was signed. And Ukraine did not comply with the agreement and fired at its Ukrainians in the Donbas.

**Comment by Ravshanovich** [comment]
1. The Black Sea Fleet has been the subject of a dispute between Russia and Ukraine since the fall of the Soviet Union in 1991.

2. In 1997, the parties finally came to an agreement and signed three documents that determined the fate of the fleet and military bases in Crimea. Two years later, in 1999, the agreements were ratified. Russia received 81.7% of the fleet, paying the Ukrainian government compensation of $526.5 million.

3. Moscow annually writes off $97.75 million of debt owed to Kiev for the right to use the water area and radar equipment of Ukraine and for possible environmental damage.

4. According to the initial agreement, the Russian Black Sea Fleet was supposed to remain in Crimea until 2017, but later this period was extended for another 25 years.

5. The 1997 agreement allows Russia to keep a 25,000-strong military contingent on the territory of Crimea, 24 artillery systems with a caliber of less than 100 mm, 132 armored vehicles and 22 military aircraft.

6. In accordance with the agreement, several Russian naval units are stationed in Sevastopol. This includes the 30th Division of surface ships. It includes the 11th brigade of anti-submarine vessels: the missile cruiser Moskva, the large anti-submarine ship Kerch and the patrol ships Inquisitive, Sharp-Witted and Ladny, as well as the 197th brigade, consisting of seven landing ships.

Also based in Sevastopol are the 41st brigade of missile boats, the 247th division of submarines, the 68th brigade of ships protecting the water area and the 422nd separate division of hydrographic vessels.

7. Moscow has two military air bases in Crimea. They are located in the villages of Kacha and Gvardeyskoye.

In November 2013, protests began in the capital of Ukraine. President Viktor Yanukovych postponed the signing of the Association agreement with the European Union. This was the reason for people to take to the streets. The action, which began with a student rally, turned into a powerful movement. Tens of thousands of people organized a tent city in the center of Kiev, began to occupy administrative buildings, burn tires. Gradually, the peaceful rally turned into a fierce confrontation between demonstrators and the police. The first victims appeared on both sides. At the same time, actions against the existing government began in the western regions of Ukraine, their heads of city and regional councils were appointed, monuments of the Soviet regime were crumbling. In February 2014, the action in Kiev, which became known as Euromaidan, reached its peak. Dozens of protesters and law enforcement officers were killed by unknown snipers. The opposition and the leaders of the protest movement carried out a coup, President Yanukovych and his family fled the country. Pro-Western leaders who are aggressively opposed to Russians, Russia, and the Soviet Union came to power. Illegal armed groups began to move from Kiev to the regions. Mass retaliatory actions against the new regime have begun in the south-east of the country.    
The crisis of the Ukrainian government in February 2014 led Crimea to the need to determine its future fate. The adoption of the new government in Ukraine meant the rupture of the historical, cultural, and social ties of the peninsula with Russia. The forces that carried out the coup in Kiev were unequivocally hostile and aggressive about Russians, including those living in Crimea.   
On March 16, a national referendum on the status of Crimea was held, according to the results of which (when more than 96% of voters voted for the republic to join the Russian Federation) the Supreme Council declared Crimea an independent sovereign state – the Republic of Crimea, within which the city of Sevastopol has a special status. And this independent state has already appealed to the Russian Federation with a proposal to accept it into the Russian Federation as a new subject. On March 18, a corresponding agreement was signed in Moscow, which entered into force on March 21 – from the date of its ratification by the Federal Assembly.

**Comment by Ravshanovich** [comment]
The annexation of Crimea to Russia took place peacefully. 
For some reason, the fascists did not send their troops to retake the Crimea, knowing that the Armed Forces of the Russian Federation were there, but sent to Donbass, where ordinary miners did not obey the new government that carried out the coup.

**Comment by Ravshanovich** [comment]
You're not my brother.
The disarmament of Ukraine, Belarus and Kazakhstan took place with the mediation of the United States. It was Washington that became the key force that prompted the young republics to abandon strategic forces.

American politicians doubted the Ukrainians' ability to control nuclear weapons and strongly opposed their proliferation in Eastern Europe.   Even there they understood that a monkey with a grenade is unpredictable.

**Comment by Ravshanovich** [comment]
в условиях тотальной коррупции перестроечного периода американцы банально боялись, что украинцы продадут ракеты кому-нибудь еще и оно ебанет по кукловодам.

**Comment by Ravshanovich** [comment]
For giving up nuclear weapons, Ukraine received generous economic preferences, as well as security guarantees enshrined in the Budapest Memorandum.

Greed outweighs

**Comment by Ravshanovich** [comment]
However, Kiev had no choice. If the country refused to transfer atomic weapons, it would be politically isolated from the West.

**Comment by Ravshanovich** [comment]
Do you still think that Ukraine is an independent state? And not a doll in the hands of the Anglo-Saxons, who ultimately need only the bowels?

**Comment by merseyV** [comment]
>the same right for the Crimeans?

Quite right ... bring back the Crimean Tartars!! 🤣

**Comment by Ravshanovich** [comment]
And here right now.  The Pindos are staging color revolutions around Russia in Georgia, Ukraine, Armenia, and tried in Belarus. The only task is to destabilize the situation around and destroy, and then rob.  It is not Russia that is expanding its bases around the United States, but on the contrary.  The United States likes to emphasize its exclusivity, commitment to peace, freedom and democracy. And under these slogans, it is not a sin to invade a foreign country, bomb the civilian population, stage another coup and plunder resources. The owners of the White House and their NATO allies are so convinced of their sanctity and exclusivity that they have long ago crossed out all norms of morality and law. 
Under various pretexts ("protection of peace, human rights and democracy", "countering the spread of communism", "restoring justice", "combating terrorism", "preventing the proliferation of weapons of mass destruction") The United States invaded sovereign states, destroyed the established order, economy, infrastructure, and brought disasters and misfortunes to these lands.

The actions of NATO aggressors in North Korea, Vietnam, the Persian Gulf, Yugoslavia, Iraq, Afghanistan, Libya, Syria and other countries have caused large-scale tragedies that have claimed millions (!) of lives. 
In addition to direct involvement in conflicts, Americans and NATO forces often act indirectly. They support internal protests and riots, train "revolutionaries", supply weapons and other assistance to the rebels.

 After the events of September 11, 2001, Americans spent about 6.5 trillion dollars on military adventures, mainly in the Middle East and Asia.

 In 73 years, the number of NATO member countries has increased 2.5 times — from 12 to 30. The maximum increase occurred after 1990, the end of the Cold War. At the very time when the USSR no longer existed, as well as the Warsaw Pact Organization, which means that there was no "Soviet threat".

 In the post-Soviet space, the United States and NATO committed or tried to prepare "color revolutions" in Georgia (2003), Ukraine (2004, 2014), Kyrgyzstan (2005), Armenia (2008, 2015), Russia (2010, 2013), Belarus (2020), Kazakhstan (2022).

The largest military operations involving the United States and NATO countries since 1949

1950-1953 . The Korean War

In 1950, U.S. Secretary of State Dean Acheson named the territory part of the American defense perimeter. Taking advantage of the boycott of the UN Security Council by the USSR, the Americans led the military intervention of UN forces on the Korean Peninsula. As a result of the war, four million Koreans died, and the peninsula was divided by the 38th parallel.

1959 The Lao Civil War

The American bombing of Laos was part of a secret CIA attempt to wrest power from a communist group allied with North Vietnam and the USSR. During the entire period of the war (1959-1975), the United States dropped about three million tons of bombs on the territory of Laos, an average of 10 tons per square kilometer and half a ton for each resident of Laos. The death toll still cannot be calculated. 

1964-1975 . The Vietnam War

Books have been written about American war crimes during this war. "Human rights defenders" not only killed thousands of civilians, raped women, burned livestock, destroyed crops, but also sprayed poison over fields — millions of tons of highly toxic chemicals. Three million residents of the country died during the war.

1989 The "Restoration of democracy" in Panama

In December 1989, the US air force attacked the cities of Panama, made landings and landed personnel and equipment. The operation lasted only five days. As a result, a regime loyal to the United States was established in Panama. On the Panamanian side, 68 soldiers and about 500 civilians were killed.

1995 Bombing of Bosnia and Herzegovina

The NATO air operation against the Bosnian Serbs "Deliberate Force" lasted 16 days. According to the latest data, the total number of deaths in the Bosnian war (1992-1995) was about 100 thousand people.

1999 The destruction of Yugoslavia

19 NATO countries participated in the military operation against Yugoslavia "Allied Force". From March 24 to June 10, 1999, alliance aircraft carried out about 2,300 missile and bomb attacks on 990 targets in Serbia and Montenegro. The total number of civilian deaths amounted to more than 2.5 thousand people, including 400 children, and about 10 thousand people were seriously injured. The Americans, with the support of their satellites, achieved the division of a single country into separate states.

2001 The invasion of Afghanistan

The war in Afghanistan has become the longest in the history of NATO. The result of the 20-year military operation Enduring Freedom was about 50,000 Afghan civilians killed, an almost completely destroyed economy, and millions of refugees. The Americans and their allies were forced to leave the country in disgrace, and the Taliban movement came to power.

2003 The invasion of Iraq

The war in Iraq turned out to be an even greater setback for the United States and its allies. Having overthrown President Saddam Hussein, American soldiers did not become heroes for the people of Iraq. According to various estimates, from 655,000 to one million people died during the military operations and occupation in Iraq. The irritation against the Americans resulted in the rise of radical Islam in the region.

2011 Air strikes on Libya

In 2011, the NATO coalition invaded Libya without the sanction of the international community, killed head of state Muammar Gaddafi and plunged the country into chaos. The civil conflict continues in the country to this day. According to various estimates, from 15 to 30 thousand people died during the armed confrontation. 

2014 Intervention in Syria

The US-led military coalition, with the active participation of Britain, Canada and other countries, intervened in the internal affairs of Syria in order to overthrow President Bashar al-Assad. According to various estimates, from 585,000 to a million Syrians died during the war.


  But fuck you all!!!

**Comment by merseyV** [comment]
@kiriles90 

Whoa!! Hang-on! You haven't left me any Nukes!!! 🤣🤣

**Comment by Ravshanovich** [comment]
> I leave you to @merseyV :p

He's stupid.

**Comment by Ravshanovich** [comment]
> So sit down, shut up, and enjoy..

The USSR collapsed because of traitors, that's a fact.  Russia has not lost yet. The rest of you will have to sit on your ass straight.

**Comment by merseyV** [comment]
[**zzzzzzzzzzzz!!**]

**Comment by merseyV** [comment]
[zzzzzzzzzzzz!!](https://i.pinimg.com/originals/1f/e9/65/1fe965e81a57c4ffc4f8bf9c2d348d6b.gif)

**Comment by merseyV** [comment]
No more boring post ... let's party!!!!

![hurray-hooray-snoopy-celebration-f2204egyx1syij2c](https://github.com/popcorn-official/popcorn-desktop/assets/137357177/5c16efac-b026-48d7-ad18-bd3ca318376b)
